sample/app/index.tsx

```diff
@@ -11,13 +11,13 @@ export default function App() {
   return (
       <View style={styles.container}>
         <FFContextProvider
-            apiKey=""
+            apiKey="" // <- add your api key for the test environment
             target={{
               identifier: 'target1', // <- replace with an identifier unique to the user, e.g. email or UUID
               name: 'target1' // <- replace with a name unique t o the user
             }}
-            async={true}
-            options={{cache: true, debug: true, streamEnabled: false, pollingEnabled: true}}
+            asyncMode={true}
+            options={{cache: true, debug: true, streamEnabled: true, pollingEnabled: true}}
```

Comment:
Some of this is the standard defaults 🤔 (e.g. polling and streaming)

Reply:
`streamEnabled` was initially set to false in the sample app.  It did need to be changed, either by setting it to `true` or as you suggest could delete the default config entirely.

Reply:
Will I change streamEnabled back to false?

Reply:
This file should be our suggestion of a good practice setup. As such, personally, what I would change from a vanilla install is to enable caching.

`asyncMode` is a case-by-case kinda thing and typically, my suggestion would be enable it only if you are ok with potentially showing defaults, or as a guard against potentially having network issues prevent initial flags being fetched. But if we're keeping it, we should drop the `={true}` part since that's not required in React.

`cache: true` I think is a no-brainer and should be on.

`debug: true` makes their app's console very noisy, so I'd only consider enabling that when needed and leave it out here.

`streamEnabled: true` we can drop since it's a default.

`pollingEnabled: true` is also superfluous since it bases itself off `streamEnabled` when not specified (when streaming is on it takes precedence, if it fails, the as long as polling hasn't been expressly _disabled_, it'll be inferred as enabled and take over)